rpi5.yaml

```diff
@@ -34,19 +34,19 @@ common_data:
   common_yocto_sources: &COMMON_YOCTO_SOURCES
     - type: git
       url: "https://git.yoctoproject.org/poky"
-      rev: "scarthgap"
+      rev: "7bb9c2255b3aed8441fba1133f350b223a3b6379"
     - type: git
       url: "https://git.openembedded.org/meta-openembedded"
-      rev: "scarthgap"
+      rev: "727811eaf256b88fd135be99559f2cbf14c82fce"
     - type: git
       url: "https://github.com/xen-troops/meta-xt-common.git"
       rev: "master"
```

Comment:
I would recommend setting a proper hash here as well because meta-xt-common is an external component for the product.

Reply:
My idea is that the `master` branch can change, for example, it may switch to the new release of yocto. And the product became non-compilable. As we have, in fact, right now with previous release tags.
So, it is better to use fixed hashes for all external components.